packages/eslint-plugin-react-hooks/__tests__/ESLintRuleExhaustiveDeps-test.js

```diff
@@ -521,7 +521,7 @@ const tests = {
       // Valid because the ref is captured.
       code: `
         function useMyThing(myRef) {
-          const myRef = useRef();
+          myRef = useRef();
```

Comment:
could you explain this change?

Reply:
(as in, to me :) I 'm not sure I understand why this const was removed.)

Reply:
`myRef` comes in as a parameter to the function. In babel 7 (and at runtime) it's a syntax error to redeclare a variable as const.

`Uncaught SyntaxError: Identifier 'myRef' has already been declared`.

This surfaces now because I've bumped `@babel/parser` as part of this PR (and `babel-eslint`, which uses that parser under the hood, is used as the parser for these eslint tests)

Reply:
The _wonderful_ error Jest outputs without this change:

![image](https://user-images.githubusercontent.com/1404810/62622522-d712c600-b91e-11e9-9dca-0ddfc25092bc.png)

Reply:
Haha I missed that it was already an arg. Thanks!

Reply:
so an alternative in this case is to remove the parameter, but I'm not sure what this test asserts TBH

Reply:
that's the reason for all the changes in the eslint tests, btw

Reply:
I checked, and this line could be removed completely. 
```suggestion
```

Reply:
perfect. the other eslint rule changes look alright?